### ISSUE TITLE
How to get code syntax highlighting for other languages?

### ISSUE THREAD

**Issue by mitesh1612** [narative/gatsby-theme-novela#410]
# Prerequisites
* [x] Have you searched the issue tracker for similar issues (either past or present)?
* [x] Can you reproduce the issue consistently?

# Description
### Version of Novela you are using
> 0.13.3

### Number of posts you have and which data source
* [x] Local - 2 for now
* [ ] Contentful

### What are you trying to achieve?
I created a post and made codeblocks using 3 `'s and chose the language to be C# (like ```csharp .. content) but when the site is generated the code syntax highlighting doesnt work for it. I also tried ```c# but even that doesn't provide me syntax highlighting. What is the correct way to get syntax highlighting for languages other than JS?

### What's blocking you?
Just want to get better syntax highlighting, by tweaking the code or adding some parameters to get proper syntax highlighting for languages like C#, Python etc for my posts would be amazing, I really love this theme and want to keep using it as is! :)  

Here are the example snaps of me encountering this issue!
![image](https://user-images.githubusercontent.com/19746697/90097172-f5da4280-dd52-11ea-8a22-f75171356e3a.png)
My markdown
![image](https://user-images.githubusercontent.com/19746697/90097172-f5da4280-dd52-11ea-8a22-f75171356e3a.png)
Generated site code block

**Comment by mitesh1612** [comment]
Update, using the markdown like this 
![image](https://user-images.githubusercontent.com/19746697/90099238-9e8aa100-dd57-11ea-8076-9e5f35e0b0cd.png)
(new markdown)
It gives me the required highlighting (as show in the next image, but shows a new error now)
![image](https://user-images.githubusercontent.com/19746697/90099289-b82be880-dd57-11ea-8370-f980d587b976.png)
(highlighted code but with an error propping up)

**Comment by brotzky** [comment]
You'll want to disable live preview on the code block (the area underneath there is trying to render what you're writing.

**Comment by mitesh1612** [comment]
@brotzky Yeah I figured that out later. Weirdly when I use `live` with csharp it gives me the Syntax highlighting but doesnt do it for normal cases. Any fixes for that?
